recipes/resiprocate/cmake/conanfile.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.53.0"
+
+
+class ResiprocateConan(ConanFile):
+    name = "resiprocate"
+    description = (
+        "The project is dedicated to maintaining a complete, correct, "
+        "and commercially usable implementation of SIP and a few related protocols."
+    )
+    license = "VSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/resiprocate/resiprocate/wiki/"
+    topics = ("sip", "voip", "communication", "signaling")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cares": [True, False],
+        "with_ssl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cares": True,
+        "with_ssl": True,
+    }
+
```

Comment:
```suggestion

    @property
    def _minimum_cpp_standard(self):
        return 11

    @property
    def _minimum_compiler_version(self):
        return {
            "gcc": "7",
            "clang": "6",
            "apple-clang": "10",
            "msvc": "191",
            "Visual Studio": "15",
        }

```
I determined the minium GCC and Clang versions by building in a container. apple-clang is based on LLVM 6 version. MSVC is just a guess based on similar recipes.